### PR TITLE
Doc formatting and improvements

### PR DESCRIPTION
**Pull request by nemchik**
# Pull request

**Purpose**
Fix mistakes, make some things a bit more consistent. Continue trying to add everything from the #overrides channel in Discord to the docs so we can remove that channel.

**Requirements**
Check all boxes as they are completed

- [x] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/apps/freshrss.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # FreshRSS
 
 [![Docker Pulls](https://img.shields.io/docker/pulls/linuxserver/duplicati?style=flat-square&color=607D8B&label=docker%20pulls&logo=docker)](https://hub.docker.com/r/linuxserver/freshrss)
-[![GitHub Stars](https://img.shields.io/github/stars/linuxserver/docker-duplicati?style=flat-square&color=607D8B&label=github%20stars&logo=github)](https://github.com/linuxserver/docker-freshrss)
+[![GitHub Stars](https://img.shields.io/github/stars/linuxserver/docker-freshrss?style=flat-square&color=607D8B&label=github%20stars&logo=github)](https://github.com/linuxserver/docker-freshrss)
```
Comment (angrycuban13) [inline docs/apps/freshrss.md:4]: Good catch lol

File: docs/overrides/fireflyiii.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# fireflyiii
```
Comment (angrycuban13) [inline docs/overrides/fireflyiii.md:1]: I think the name should be `Firefly III`

File: docs/overrides/dashmachine.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# dashmachine
```
Comment (angrycuban13) [inline docs/overrides/dashmachine.md:1]: Shouldn't we capitalize `dashmachine` to `Dashmachine` to match everything else?

File: docs/overrides/autoscan.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# autoscan
```
Comment (angrycuban13) [inline docs/overrides/autoscan.md:1]: Shouldn't we capitalize `autoscan` to `Autoscan` to match everything else?

File: docs/overrides/bw_plex.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# bw_plex
```
Comment (angrycuban13) [inline docs/overrides/bw_plex.md:1]: Shouldn't we capitalize `bw_plex` to `Bw_plex` or maybe `BW_Plex` to match everything else?

File: docs/overrides/kitana.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# kitana
```
Comment (angrycuban13) [inline docs/overrides/kitana.md:1]: Shouldn't we capitalize `kitana` to `Kitana` to match everything else?

**Review by angrycuban13 — CHANGES_REQUESTED** [review]

File: docs/overrides/bw_plex.md (hunk 5 above)
Comment (nemchik) [reply docs/overrides/bw_plex.md:1]: This one no actually. The repo where it came from always has it formatted as `bw_plex` so I think they intended it to be lowercase.

**Review by angrycuban13 — APPROVED** [review]

**Pull request merged**